### DISCUSSION TITLE
Error  :     > Mesh requires second-order elements.

### DISCUSSION THREAD

**Discussion by akihoo** [neperfepx/FEPX#74] [Q&A] [answered]
I am trying to simulate a uniaxial tension case based on the `1_uniaxial` example. The only difference is that I use hex elements as below, then I got this error message `Error : > Mesh requires second-order elements.`. 

The mesh is generated by this command `neper -M n100-id1.tess -order 2 -elt hex`. It seems the mesh is 2-order. Why does this error happen?

![img](https://user-images.githubusercontent.com/14359322/156525398-3458ae3b-c147-41bb-8f55-b2e3f9a76eb4.png)

The error message is
```
==========================     F   E   P   X   ==========================
Info   : A finite element software package for  polycrystal plasticity.
Info   : Version 1.2.1
Info   : Running on 1 cores.
Info   : <https://fepx.info>
Info   : Copyright (C) 1996-2021, DPLab, ACME Lab.
Info   : ---------------------------------------------------------------
Info   : Start time: 2022-3-3 at 16:16
Info   : Loading simulation...
Info   :   [i] Parsing file `simulation.config'...
Error  :     > Mesh requires second-order elements.
========================================================================
```

**Comment by rquey** [comment, marked as answer]
The error message could be improved, but FEPX simply does not support hex elements. It only supports second-order tet elements.
